README.md

```diff
@@ -0,0 +1,60 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Review Past Project
+
+> A project to microverse practice Code Review 2.0 training.
+
+## Built With
+
+- HTML
+- CSS
+- VS-Code
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] It is good that you've added the live demo section to your Readme. However, the live demo link seems to be broken. Kindly fix it.

Reply:
It will be very nice if you add some emojis for your inline comments and reviews, emojis always make your review more human-friendly and make you look like a kind person. :smile:  :hugs: .